test/test_core.py

```diff
@@ -9824,7 +9824,7 @@ def test_fcoverage_mapping(self):
 '''
     self.set_setting('NODERAWFS')
     self.set_setting('EXIT_RUNTIME')
-    self.do_core_test('test_hello_world.c', cflags=['-fprofile-instr-generate', '-fcoverage-mapping', '-g'])
+    self.do_core_test('test_hello_world.c', cflags=['-fprofile-instr-generate', '-fcoverage-mapping', '-g', '-lstubs'])
```

Comment:
It looks like this is due to `uname` reference which is coming from `system/lib/compiler-rt/lib/profile/InstrProfilingUtil.c`.

Can you instead remove `COMPILER_RT_HAS_UNAME` from `tools/system_libs.py` and add `|| defined(__EMSCRIPTEN__)` to line 57 of InstrProfilingPort.h?

Reply:
Nice, that's a better fix.